packages/coil-extension/src/background/services/multipleInstanceDetector.ts

```diff
@@ -99,6 +107,19 @@ export class MultipleInstanceDetector {
     )
   }
 
+  externalMessageListener() {
+    this.wextApi.runtime.onMessageExternal.addListener(function (
+      message: ExtensionMessage,
+      sender: any,
```

Comment:
1. Use an arrow function 
2. Don't use `any` (check the rest of the code for the Sender type) 
3. You shouldn't need the `TResult`, just use `void`

Reply:
Type inference SHOULD work here though 
I need to play with it